bin/cml-send-comment.js

```diff
@@ -7,7 +7,7 @@ const yargs = require('yargs');
 
 const { head_sha: HEAD_SHA, handle_error, comment } = process.env.GITHUB_ACTIONS
   ? require('../src/github')
-  : require('../src/gitlab');
+  : require('../src/bitbucket');
```

Comment:
we have to find a way to determine the env. However this might be useless with the new approach